setup.py

```diff
@@ -6,21 +6,27 @@
 # file, You can obtain one at http://mozilla.org/MPL/2.0/.
 from setuptools import setup, find_packages
 
-tests_require = ['mock', 'pyspark', 'pytest', 'pytest-cov', 'pytest-timeout']
+tests_require = ["mock", "pyspark", "pytest", "pytest-cov", "pytest-timeout"]
```

Comment:
Oh, sorry for the quote changes; I ran `black` over it and didn't look at the output.

Reply:
No problem. I consider it good if this matches the black syntax but it is excluded in my `bin/format-python` script. If you'd prefer we can remove it from this PR and I can push the change myself.

Reply:
I reverted and made the minimal change.